### PR TITLE
Make changes to docker-compose for CI compatibility and add edgex-secrets-setup logging

### LINKED ISSUE #342: Update the blackbox-testing docker-compose file
Update the blackbox-testing docker-compose file to reflect the changes made by https://github.com/edgexfoundry/developer-scripts/pull/184

### PR DESCRIPTION
**Pull request by bnevis-i**
Output edgex-secrets-setup logs and fix docker-compose issues on CI

Add logging of edgex-secrets-setup.
Fix issue with SELinux on the Jenkins CI machine
preventing security secrets from being shared
across docker containers (manifesting as
access denied when trying to write files on
shared volumes).

See http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/ for more explanation.

Fixes #342

Signed-off-by: Bryon Nevis <bryon.nevis@intel.com>

### CODE REVIEW

**Review by michaelestrin — DISMISSED** [review]
LGTM

**Comment by bnevis-i** [comment]
recheck

**Comment by bnevis-i** [comment]
Despite the fact the bot can't update the github status, PR reveals blackbox-test docker-compose is failing at 
```
level=INFO ts=2019-12-17T23:25:27.293244085Z app=edgex-security-secrets-setup source=generate.go:230 msg="pki-init generation completes"
level=ERROR ts=2019-12-17T23:25:27.304057162Z app=edgex-security-secrets-setup source=init.go:87 msg="open /tmp/edgex/secrets/ca/ca.pem: permission denied"
```

**Comment by bnevis-i** [comment]
@michaelestrin   Seems to be healthier now, but still not a complete pass.  Security services seem to start now.  Check out these logs: https://jenkins.edgexfoundry.org/job/blackbox-verify-security-centos7-blackbox-4c-2g-master/270/consoleText

File: docker-compose.yml [hunk 1]
```diff
@@ -69,9 +69,9 @@ services:
       - consul-data:/consul/data
       - consul-scripts:/consul/scripts
       - vault-config:/vault/config
-      - /tmp/edgex/secrets/ca:/tmp/edgex/secrets/ca:ro
-      - /tmp/edgex/secrets/edgex-vault:/tmp/edgex/secrets/edgex-vault:ro
-      - /tmp/edgex/secrets/edgex-kong:/tmp/edgex/secrets/edgex-kong:ro
+      - /tmp/edgex/secrets/ca:/tmp/edgex/secrets/ca:ro,z
+      - /tmp/edgex/secrets/edgex-vault:/tmp/edgex/secrets/edgex-vault:ro,z
+      - /tmp/edgex/secrets/edgex-kong:/tmp/edgex/secrets/edgex-kong:ro,z
```
Comment (bnevis-i) [inline docker-compose.yml:74]: See http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/ for a description of what is going on here.

**Comment by michaelestrin** [comment]
> Seems to be healthier now, but still not a complete pass. Security services seem to start now. Check out these logs: https://jenkins.edgexfoundry.org/job/blackbox-verify-security-centos7-blackbox-4c-2g-master/270/consoleText

Looks like core services are staying up after they're started.  Tests are not actually running; everything fails before then.  Looks rules engine related.

image

**Review by michaelestrin — APPROVED** [review]

**Comment by bnevis-i** [comment]
FYI: Collin's recommendation is that on an SELinux system the 'z' flag should be specified for all shared resources between host/container and container/container.  I plan to wait until the CI issues are resolved and then apply this fix universally for the entire docker-compose.

**Comment by eb-oss** [comment]
recheck

**Comment by eb-oss** [comment]
recheck

**Comment by eb-oss** [comment]
recheck

**Pull request merged**